### ISSUE TITLE
Multiple replace - Apply doesn't seems to do anything ...

### ISSUE THREAD

**Issue by Sopor** [SubtitleEdit/subtitleedit#6633]
If i use `Multiple replace` and add rules and I press `Apply` all the matching lines disappear and when i press cancel no text have been replaced. I press cancel because i have already applied my changes, so if i want to leave without replacing a second time i press cancel.

If i now open `Multiple replace` again all the rules have disappeared.

For it to work i need to press OK, but then i wondering, what will `Apply` do?

Apply for me is that it will execute the changes i have made, but without closing the window, so i can do a second replace.

In Windows you have OK and Apply. When you are finished, you click on OK and not Apply and then OK.
You use Apply when you want to save your changes without closing the window.

Subtitle Edit 3.6.11

**Comment by jeesusonherraturku** [comment]
I made an email report of this a while ago. Here is my shortened email:

"multiple lines->replace function. 

 This way I was able to set my desired position to all lines by replacing the faulty coordinates with desired ones. 


However, I think the apply button is not working properly in that window. 

Even without pressing Apply it already shows before and after, when you type on the search and replace fields and press update.

Usually when you press Apply in an app it updates the text in the main window, but in this case it generates new update to the  before- after section.  You still need to press OK to generate the change to the main window. Maybe this is desired functionality, but it was little bit confusing. "

Subtitle Edit is a great app, so this is a minor issue in my opinion.
